ClosedXML.Tests/ClosedXML.Tests.csproj

```diff
@@ -22,11 +22,11 @@
 
 
 	<ItemGroup>
-		<PackageReference Include="Microsoft.NET.Test.Sdk" Version="17.1.0" />
+		<PackageReference Include="Microsoft.NET.Test.Sdk" Version="17.2.0" />
```

Comment:
I would prefer a wildcard 17.* in order to not have to check this line every so often

Reply:
I am on the !prevent risks side" of "Benefit of reduced effort preventing upgrades" vs  "risk to break anything"... 
packages used in production code get updated using dependabot
dev time dependecies are blacklisted independabot to prevent noise and can be upgraded by any dev